test/auth/auth.e2e-spec.ts

```diff
@@ -68,6 +71,87 @@ describe('AuthController (e2e)', () => {
     });
   });
 
+  describe('login the registered user', () => {
+    const loginInput: LoginInput = {
+      username: 'e2etester@random.com',
+      password: '12345678',
+    };
+
+    it('should succesfully login the user', () => {
+      return request(app.getHttpServer())
+        .post('/auth/login')
+        .send(loginInput)
+        .expect(HttpStatus.OK)
+        .expect((res) => {
+          const token = res.body;
+          expect(token).toHaveProperty('accessToken');
+          expect(token).toHaveProperty('refreshToken');
+        });
+    });
+
+    it('should failed to login with wrong credential', () => {
+      return request(app.getHttpServer())
+        .post('/auth/login')
+        .send({ ...loginInput, password: 'wrong-passs' })
+        .expect(HttpStatus.UNAUTHORIZED);
+    });
+  });
+
+  describe('login the registered user', () => {
```

Comment:
I think this test block got duplicated by mistake? Could you check it once. :pray:

Reply:
oppss.. removed the chunk